uci-uav-forge/imaging_training_2024
# Refactor and Parameterize Isaac-YOLO Conversion

## Description
- Refactor the YOLO formatting code to be more readable and use OOP.
- Parameterize selection of class category (e.g., shapes only).

## Testing
- Run the data generation code to see if only one class is produced per target, and that it's accurate.
- Use the code in `draw_yolo_bbox.py` to see that the tiles and associated labels are correct.

## References
https://www.reddit.com/r/ProgrammerHumor/comments/tk9fd3/the_30_yearold_ooper
